src/Cache/ContaoCacheWarmer.php

```diff
@@ -116,8 +116,17 @@ private function generateConfigCache($cacheDir)
     {
         $dumper = new CombinedFileDumper($this->filesystem, new PhpFileLoader(), $cacheDir.'/contao', true);
 
-        $dumper->dump($this->locator->locate('config/autoload.php', null, false), 'config/autoload.php');
-        $dumper->dump($this->locator->locate('config/config.php', null, false), 'config/config.php');
+        try {
+            $dumper->dump($this->locator->locate('config/autoload.php', null, false), 'config/autoload.php');
+        } catch (\InvalidArgumentException $e) {
+            // Ignore, no autoload.php existed in any bundle
+        }
+
+        try {
+            $dumper->dump($this->locator->locate('config/config.php', null, false), 'config/config.php');
```

Comment:
That's correct. We also did not add a try-catch because the core bundle supplied an `autoload.php`. I guess it's safe to add it as well, we never know about future changes ;-)